Yoast/wordpress-seo
# P3-60 Add basic SEMrush modal container

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Adds a basic container for the SEMrush modal based and a Slot/Fill implementation for the modal content.

## Relevant technical choices:

* Note that for now I passed some props to the `KeyphrasesTable` component just for the sake of making ESLint happy. This part should be reviewed during future iterations. It's probably fine, as these props will be likely used within the table component.

## Test instructions

This PR can be tested by following these steps:

- build
- edit a post
- enter a focus keyphrase and click the "Get related keyphrases" button
- the modal dialog opens
- check all the component in the content render correctly
- in the `RelatedKeyphraseModalContent` change the `isLoading` prop default value to `false`
- build
- refresh and reopen the modal
- observe the `SemRushLoading` content is _not_ rendered

## UI changes

## Documentation

## Quality assurance


Fixes [P3-60]
